cli/src/test/java/io/specmesh/cli/StorageConsumptionFunctionalTest.java

```diff
@@ -89,14 +88,15 @@ class StorageConsumptionFunctionalTest {
     @Test
     void shouldGetStorageAndConsumptionMetrics() throws Exception {
 
-        Provisioner.provision(
-                        true,
-                        false,
-                        false,
-                        API_SPEC,
-                        "./build/resources/test",
-                        KAFKA_ENV.adminClient(),
-                        Optional.of(KAFKA_ENV.srClient()))
+        Provisioner.builder()
```

Comment:
Existing use of `Provisioner.provision` is updated to use the new builder pattern.